ext/standard/crc32.c

```diff
@@ -15,10 +15,15 @@
 */
 
 #include "php.h"
+#ifdef PHP_WIN32
+#include "config.w32.h"
+#else
+#include <php_config.h>
+#endif
```

Comment:
Is this part of the change needed? These should be pulled in by php.h.

Reply:
Did not find these includes in php.h on first sight..., but copied these lines from ext/standard/html.c...., but you are right, php_config.h/config.w32.h is sourced via php.h --> php_compat.h, will remove these lines..., thanks for review!